src/redux/services/resource.ts

```diff
@@ -457,8 +458,17 @@ export function reprocessResources(
         resource.name = `Patch: ${resource.name}`;
       }
 
-      resource.kind = resource.content.kind;
-      resource.version = resource.content.apiVersion;
+      const isKustomziationFile = resource.filePath.endsWith('kustomization.yaml');
```

Comment:
let's have a KUSTOMIZATION_FILE_NAME constant here and use it in the `isKustomizationFile` from `kustomize.ts` as well

Reply:
good idea!